varia/sass/components/header/_site-mobile-nav-side.scss

```diff
@@ -0,0 +1,107 @@
+// Updates the Mobile Navigation to be next to the site title.
+@include media(mobile-only) {
+	.mobile-nav-side {
+		.has-menu {
+			&:before,
+			&:after {
+				content: none;
+			}
+			position: relative;
+			display: grid;
+			grid-gap: 2px;
+			grid-template-areas: 
+				"site-logo site-logo" 
+				"social-navigation main-navigation";
+
+			&.has-title-and-tagline {
+				grid-template-areas: 
+					"site-logo site-logo" 
+					"site-title main-navigation" 
+					"site-description main-navigation" 
+					"social-navigation social-navigation";
+
+				.site-title {
+					grid-area: site-title;
+					align-self: end;
+				}
+
+				.site-description {
+					grid-area: site-description;
+					align-self: start;
+				}
+			}
+
+			& > *:not(.site-logo) {
+				margin: 0;
+			}
+
+			.site-logo {
+				grid-area: site-logo;
+				justify-self: center;
+				margin: $baseline-unit 0;
+			}
+
+			.social-navigation {
+				grid-area: social-navigation;
+				align-self: center;
+			}
+
+			.main-navigation {
+				grid-area: main-navigation;
+				//align-self: center;
+			}
+		}
+
+		.site-header #site-navigation.main-navigation {
+			margin: 0;
+			justify-self: end;
+			//text-align: right;
+
+			#toggle-menu {
+				grid-area: menu-toggle;
+			}
+
+			#toggle-cart[for="woocommerce-toggle"] {
+				grid-area: cart-toggle;
+			}
+
+			#toggle-cart[for="woocommerce-toggle"].button,
+			#toggle-menu {
+				line-height: 1.5;
+				padding: (1.5 * $baseline-unit);
+				display: inline-block;
+				text-align: center;
+				justify-self: end;
+				font-size: #{map-deep-get($config-global, "font", "size", "xs" )};
+			}
+			#toggle-cart[for="woocommerce-toggle"] {
+				margin-top: $baseline-unit;
+			}
+			#toggle-cart[for="woocommerce-toggle"].button svg {
+				margin-top: -2px;
+			}
+
+			.main-menu-container {
```

Comment:
I don't think we can use class names here as they change for each menu. The way we do it in other themes is with a `>div` selector.

Reply:
Changed it